notification.c

```diff
@@ -167,12 +167,13 @@ struct mako_notification *get_tagged_notification(struct mako_state *state,
 }
 
 void close_group_notifications(struct mako_notification *top_notif,
-	       enum mako_notification_close_reason reason) {
+	       enum mako_notification_close_reason reason,
+			   bool add_to_history) {
```

Comment:
One last detail: the indentation here is a bit off, we use 2 tabs for function arguments.